common/messaging/kafkaClient.go

```diff
@@ -65,3 +68,16 @@ func (c *kafkaClient) NewConsumer(topicName, consumerName string, concurrency in
 	consumer, err := c.client.NewConsumer(consumerConfig)
```

Comment:
Fixing the interface issue with the latest kafka client as part of this?

Reply:
I will take care of it in a separate PR.